tests/php/Extension/FluentBadgeExtensionTest.php

```diff
@@ -44,12 +44,14 @@ protected function setUp()
         // Clear cache
         Locale::clearCached();
 
-        FluentState::singleton()->setLocale('en_NZ');
+        FluentState::singleton()->withState(function (FluentState $newState) {
+            $newState->setLocale('en_NZ');
 
-        $this->mockPage = $this->objFromFixture(SiteTree::class, 'test_page');
-        $this->mockController = new FluentStubController($this->mockPage->ID);
-        $this->extension = new FluentBadgeExtension();
-        $this->extension->setOwner($this->mockController);
+            $this->mockPage = $this->objFromFixture(SiteTree::class, 'test_page');
+            $this->mockController = new FluentStubController($this->mockPage->ID);
+            $this->extension = new FluentBadgeExtension();
+            $this->extension->setOwner($this->mockController);
+        });
```

Comment:
My only thought here is that this would mean that the state would be reset back to what it was after this line (before any test actually runs). Perhaps the setUp method should be left the way it was? The tests are green though so it'd only be a semantic/code comprehension issue